.github/workflows/ndc-nodejs-lambda-connector.yaml

```diff
@@ -94,9 +94,19 @@ jobs:
         with:
           images: ${{ env.DOCKER_REGISTRY }}/${{ env.DOCKER_IMAGE_NAME }}
 
+      - name: Get npm package version
+        id: get-npm-package-version
+        run: |
+          PACKAGE_VERSION=`npm version | sed -rn "2 s/.*: '([^']*)'.*/\1/g; 2 p"`
```

Comment:
What is the version of? The `npm` binary itself?

Reply:
Or does this return the version field from `package.json`?

Reply:
This gets the version field from package.json.